base/reflection.jl

```diff
@@ -417,7 +417,28 @@ julia> isimmutable([1,2])
 false
 ```
 """
-isimmutable(@nospecialize(x)) = (@_pure_meta; !typeof(x).mutable)
+function isimmutable(@nospecialize(x))
```

Comment:
We generally don't start introducing "live" depwarns of exported functions in the middle of a stable release. Instead you should move this to the end of `base/deprecated.jl` but don't actually issue the depwarn. Most or all of the "live" depwarns there are for types or methods that were never exported in the first place and therefore not protected by the "no breaking changes" rule. Thus, those deprecations are not necessary but are there as a kindness. In contrast, this is part of Julia's public API.

Reply:
ok! moved `isimmutable` from `reflections.jl` to the bottom of `deprecated.jl` in 6b76c59